tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932237.yaml

```diff
@@ -42,7 +42,7 @@ tests:
             method: "GET"
             port: 80
             headers:
-              User-Agent: set
+              User-Agent: "set help"
```

Comment:
Use a command that works more commonly (bash / POSIX), e.g., `set -o history`.

Reply:
regression tests are failing again with this, I assume that's because of what's described in #3401?

Reply:
Yes. I've added a dependency to that issue. We need to resolve that first.